rflx/specification/integration.py

```diff
@@ -0,0 +1,137 @@
+from pathlib import Path
```

Comment:
I find it a bit odd to have integration inside `rflx.specification`, as it is not part of the specification, but some separate information. How about moving `integration` into `rflx`?

Reply:
I moved it to `rflx`